server.js

```diff
@@ -29,7 +29,13 @@ app.post('/user/register', async (req, res) => {
 		console.log('failed');
 	}
 	const {username,password,firstName,lastName,dateOfBirth,email,telephone,gender} = req.body;
-
+const userCheck=userModel.findOne({username,password,firstName,lastName,dateOfBirth,email,telephone,gender},(err,doc){
```

Comment:
The validation should be lookinh into only one property. for example the username.